packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -675,6 +748,29 @@ class Scaffold extends StatefulWidget {
     return scaffoldScope.geometryNotifier;
   }
 
+  /// Sets the [ScaffoldGeometry.floatingActionButtonNotchMaker] for the closest
+  /// [Scaffold] ancestor of the given context if one exist.
+  ///
+  /// It is guaranteed that `notchMaker` will only be used for making notches
+  /// in the top edge of the [BottomAppBar], the start and end offsets given to
```

Comment:
rather than linking to [BottomAppBar], I think it would be strictly correct to link to [bottomNavigationBar].

Reply:
done